docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.3"
 services:
   web:
-    image: unleashorg/unleash-server:4.12.6
+    image: unleashorg/unleash-server:4.16.0
```

Comment:
This version adds constraints.

See https://docs.getunleash.io/how-to/how-to-add-strategy-constraints